data/views/node.2.0.json

```diff
@@ -50,5 +50,19 @@
     <% } %>
     <% } %>
     "type": "<%=type%>",
-    "workflows": "<%=basepath %>/nodes/<%=id%>/workflows" 
+    "workflows": "<%=basepath %>/nodes/<%=id%>/workflows",
+    <% if (ibms === null) { %>
```

Comment:
Since this has been tested with these changes, i plan to keep them as is for now. Will change them in the snmp story.